n2y/notion.py

```diff
@@ -319,6 +321,14 @@ def get_child_notion_blocks(self, block_id):
             else:
                 params["start_cursor"] = data["next_cursor"]
 
+    def get_comments(self, block_id):
+        # TODO: Make pagination work. Currently only pulls first 100 comments.
```

Comment:
Nice! If this takes less than 30 min to do, let's just do it. Otherwise, I'd raise a `NotImplementedException` if we see that there are more comments.